File: place-service/src/main/java/pl/edu/pg/benchmarking/place/dto/GetPlaceResponse.java
package pl.edu.pg.benchmarking.place.dto;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Getter;
import pl.edu.pg.benchmarking.place.Place;

import java.util.function.Function;

@Getter
@AllArgsConstructor(access = AccessLevel.PRIVATE)
public class GetPlaceResponse {

    private Integer id;

    private String city;

    private String coordinates;

    public static Function<Place, GetPlaceResponse> entityToDtoMapper() {
        return place -> new GetPlaceResponse(
                place.getId(), place.getCity(), place.getCoordinates());
    }
}

File: monolith/website/place/dto/get_place_response.py
from website.place.place import Place
from static.utils import attributes


def entity_to_dto_mapper(place: Place):
    return GetPlaceResponse(place.id, place.city, place.coordinates)


class GetPlaceResponse:
    def __init__(self, id: int, city: str, coordinates: str) -> None:
        self.id = id
        self.city = city
        self.coordinates = coordinates

    def as_dict(self):
        return attributes(self)